README.md

```diff
@@ -1,11 +1,12 @@
-# CakePHP Framework
-
+<p align="center"><a href="https://cakephp.org" target="_blank"><img src="https://cakephp.org/img/trademarks/logo-1.jpg"></a></p>
```

Comment:
If you navigate to the cakephp homepage on the top left and right click the image, you have it. It is also above in my comment: `https://cakephp.org/img/cake-logo.svg` - it would just need to be inverted.

@phpnut does an inverted version of https://cakephp.org/img/cake-logo.svg exist? Could it be uploaded to https://cakephp.org/img/cake-logo-inverted.svg or https://cakephp.org/img/cake-logo-invert.svg ? If you want me to I can invert it using the brand color and send you the svg file.